features/gba-dev/fields-mapping-gba.csv

```diff
@@ -72,7 +72,7 @@ verblijfplaatsBinnenland.verblijfadres.huisnummertoevoeging,verblijfplaats.straa
 verblijfplaatsBinnenland.verblijfadres.korteNaam,verblijfplaats.straat|verblijfplaats.locatiebeschrijving
 verblijfplaatsBinnenland.verblijfadres.locatiebeschrijving,verblijfplaats.straat|verblijfplaats.locatiebeschrijving
 verblijfplaatsBinnenland.verblijfadres.postcode,verblijfplaats.straat|verblijfplaats.postcode|verblijfplaats.locatiebeschrijving
-verblijfplaatsBinnenland.verblijfadres.straat,verblijfplaats.straat|verblijfplaats.naamOpenbareRuimte|verblijfplaats.locatiebeschrijving
+verblijfplaatsBinnenland.verblijfadres.straat,verblijfplaats.naamOpenbareRuimte|verblijfplaats.locatiebeschrijving
```

Comment:
ook hier is verblijfplaats.straat ten onrechte verwijderd, omdat die nodig is voor het bepalen van verblijfplaats.type

Reply:
Aangepast